pkg/bootstrap/docker/host/host.go

```diff
@@ -76,155 +72,45 @@ func (h *HostHelper) EnsureVolumeShare() error {
 	return nil
 }
 
-func catHostFile(hostFile string) string {
-	file := path.Join("/rootfs", hostFile)
-	return fmt.Sprintf("cat %s", file)
-}
-
-// CopyFromHost copies a set of files from the Docker host to the local file system
-func (h *HostHelper) CopyFromHost(sourceDir, destDir string) error {
+// DownloadDirFromContainer copies a set of files from the Docker host to the local file system
+func (h *HostHelper) DownloadDirFromContainer(sourceDir, destDir string) error {
```

Comment:
I was hoping to leave these names more broad to signpost a single path for down/uploading arbitrary files from the container, if required in the future for whatever reason.  If necessary this may entail adding in the other bind mounts up top, but I left those out for now.  What's best to do here?